packages/intent-screen/src/index.ts

```diff
@@ -0,0 +1,4 @@
+export { default } from './components/intent-screen';
+export { default as SelectItems } from './components/intent-screen-select-items';
```

Comment:
Hmmm...I think this might be a little weird as it's used by any step of flow and does not belong to the intent screen

Reply:
Oh that's an interesting point! Maybe we should follow up and rename this package to `onboarding-components`. WDYT?